src/data/reducers/roles.js

```diff
@@ -1,7 +1,7 @@
 import actions from '../actions/roles';
 
 const initialState = {
-  canUserViewGradebook: null,
+  canUserViewGradebook: true,
```

Comment:
why?

Reply:
because there were complaints that the recent update caused a brief flash of "you are not allowed" until the roles fetch came back.  This way, you default to "fetching" which makes more sense anyway.